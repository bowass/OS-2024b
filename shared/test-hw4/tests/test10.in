59477
31315
19203
53199
60921
50687
34209
53360
43362
41469
36318
31167
30374
53875
22330
61326
8972
46816
40685
46921
31306
43045
36407
55796
58971
37705
21838
9020
20529
52711
33993
41140
33901
65241
26891
52090
3029
54936
47840
10283
8862
62720
34142
7329
33531
33227
57579
35242
21702
954
63370
1028
37235
21256
58972
65121
48304
30198
23745
41524
51420
48341
32629
33582
4517
423
63158
3830
32112
34945
57031
38389
29644
7283
18816
65189
53159
48725
2819
7316
45229
26859
28281
36161
1745
4553
55448
13617
64146
14228
36093
52193
15926
21548
19902
3676
32021
36948
43797
63781
47225
25355
44047
44382
60739
37563
46092
32587
44140
32113
40500
27863
55294
36441
47838
33025
50463
728
30826
43328
39683
575
40722
55190
37196
16567
36893
61892
36786
6018
55927
10444
42041
42158
12169
36641
15207
17683
44230
47933
42312
43014
11727
17732
63322
23746
29233
24405
4316
58993
62532
29568
40658
56189
22887
52321
64502
52614
50618
52889
2894
51689
52127
15957
22586
44725
11101
17499
3209
48405
30712
14340
18766
25687
55686
28535
38078
11410
26723
2712
54547
60563
52482
16837
3697
41621
4049
15362
32373
46155
42211
26697
38549
691
16794
14989
4782
20273
48410
27054
61800
64168
35268
7454
42098
51019
8237
53675
5385
5823
57084
64824
61300
34595
4193
11358
64895
44773
54698
17067
10202
40715
40169
18061
7599
682
40785
40259
32870
64533
3252
40644
37162
28839
10086
12643
4096
18128
5193
62532
27742
11454
29155
45679
38794
52685
56512
27729
34747
15739
8001
47345
18200
34690
63149
42808
42806
54076
45194
1675
28080
55524
1659
20916
27473
59387
29711
27518
41764
20273
11420
44192
36317
53909
44224
4685
62376
14381
4935
37949
29773
24450
21279
64825
11740
37540
53462
41725
47519
26269
18752
30904
58959
11618
45050
44750
56911
44821
41290
12439
9876
10311
34920
14798
28616
49427
42256
28246
48416
8427
24714
59850
25832
62515
35256
34845
2176
45167
32142
51243
33637
39044
62262
22874
64733
61018
24974
42469
8493
42422
16481
61213
16461
54862
39959
39867
26874
37458
62505
57022
32447
19549
40869
22668
18713
26956
44067
13173
36231
19124
16457
63330
11768
30657
63566
41946
46200
2144
51939
56364
37743
6516
47890
2766
59402
33000
43884
28740
45466
5454
8578
12923
11315
24300
42552
63952
399
18628
6248
16595
17138
63984
36659
55676
36609
60131
10420
3905
54962
31577
19313
26418
19738
58456
24602
7513
16415
34419
48562
2859
55524
45383
49594
13168
63156
10903
33382
26189
61733
52599
26462
16281
59800
23527
21981
2503
37184
30254
45343
28779
35911
8311
40707
18938
7301
30257
21294
13072
60140
49800
22196
3473
48797
8622
49240
39043
33281
38803
22216
48877
45229
27146
15137
44034
27153
64447
8195
62404
34976
44791
58632
14443
44925
17491
11581
23367
43871
49397
26417
9472
61130
40531
19622
51608
9648
60434
51951
14671
13219
8977
44144
27675
2478
63547
10818
59516
31716
45256
53951
55672
41786
39608
41137
48001
19008
59793
6753
33712
15251
36994
63937
36786
30322
10729
16900
16484
44335
27190
52454
8250