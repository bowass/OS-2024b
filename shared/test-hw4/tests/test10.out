Virtual address: 59477 Physical address: 85 Value: 0
Virtual address: 31315 Physical address: 339 Value: -108
Virtual address: 19203 Physical address: 515 Value: -64
Virtual address: 53199 Physical address: 975 Value: -13
Virtual address: 60921 Physical address: 1273 Value: 0
Virtual address: 50687 Physical address: 1535 Value: 127
Virtual address: 34209 Physical address: 1697 Value: 0
Virtual address: 53360 Physical address: 1904 Value: 0
Virtual address: 43362 Physical address: 2146 Value: 42
Virtual address: 41469 Physical address: 2557 Value: 0
Virtual address: 36318 Physical address: 2782 Value: 35
Virtual address: 31167 Physical address: 3007 Value: 111
Virtual address: 30374 Physical address: 3238 Value: 29
Virtual address: 53875 Physical address: 3443 Value: -100
Virtual address: 22330 Physical address: 3642 Value: 21
Virtual address: 61326 Physical address: 3982 Value: 59
Virtual address: 8972 Physical address: 4108 Value: 0
Virtual address: 46816 Physical address: 4576 Value: 0
Virtual address: 40685 Physical address: 4845 Value: 0
Virtual address: 46921 Physical address: 4937 Value: 0
Virtual address: 31306 Physical address: 330 Value: 30
Virtual address: 43045 Physical address: 5157 Value: 0
Virtual address: 36407 Physical address: 5431 Value: -115
Virtual address: 55796 Physical address: 5876 Value: 0
Virtual address: 58971 Physical address: 5979 Value: -106
Virtual address: 37705 Physical address: 6217 Value: 0
Virtual address: 21838 Physical address: 6478 Value: 21
Virtual address: 9020 Physical address: 4156 Value: 0
Virtual address: 20529 Physical address: 6705 Value: 0
Virtual address: 52711 Physical address: 7143 Value: 121
Virtual address: 33993 Physical address: 7369 Value: 0
Virtual address: 41140 Physical address: 7604 Value: 0
Virtual address: 33901 Physical address: 7277 Value: 0
Virtual address: 65241 Physical address: 7897 Value: 0
Virtual address: 26891 Physical address: 7947 Value: 66
Virtual address: 52090 Physical address: 8314 Value: 50
Virtual address: 3029 Physical address: 8661 Value: 0
Virtual address: 54936 Physical address: 8856 Value: 0
Virtual address: 47840 Physical address: 9184 Value: 0
Virtual address: 10283 Physical address: 9259 Value: 10
Virtual address: 8862 Physical address: 9630 Value: 8
Virtual address: 62720 Physical address: 9728 Value: 0
Virtual address: 34142 Physical address: 1630 Value: 33
Virtual address: 7329 Physical address: 10145 Value: 0
Virtual address: 33531 Physical address: 10491 Value: -66
Virtual address: 33227 Physical address: 10699 Value: 114
Virtual address: 57579 Physical address: 10987 Value: 58
Virtual address: 35242 Physical address: 11178 Value: 34
Virtual address: 21702 Physical address: 11462 Value: 21
Virtual address: 954 Physical address: 11706 Value: 0
Virtual address: 63370 Physical address: 11914 Value: 61
Virtual address: 1028 Physical address: 12036 Value: 0
Virtual address: 37235 Physical address: 12403 Value: 92
Virtual address: 21256 Physical address: 12552 Value: 0
Virtual address: 58972 Physical address: 5980 Value: 0
Virtual address: 65121 Physical address: 7777 Value: 0
Virtual address: 48304 Physical address: 12976 Value: 0
Virtual address: 30198 Physical address: 13302 Value: 29
Virtual address: 23745 Physical address: 13505 Value: 0
Virtual address: 41524 Physical address: 13620 Value: 0
Virtual address: 51420 Physical address: 14044 Value: 0
Virtual address: 48341 Physical address: 13013 Value: 0
Virtual address: 32629 Physical address: 14197 Value: 0
Virtual address: 33582 Physical address: 14382 Value: 32
Virtual address: 4517 Physical address: 14757 Value: 0
Virtual address: 423 Physical address: 15015 Value: 105
Virtual address: 63158 Physical address: 15286 Value: 61
Virtual address: 3830 Physical address: 15606 Value: 3
Virtual address: 32112 Physical address: 15728 Value: 0
Virtual address: 34945 Physical address: 16001 Value: 0
Virtual address: 57031 Physical address: 16327 Value: -79
Virtual address: 38389 Physical address: 16629 Value: 0
Virtual address: 29644 Physical address: 16844 Value: 0
Virtual address: 7283 Physical address: 10099 Value: 28
Virtual address: 18816 Physical address: 17024 Value: 0
Virtual address: 65189 Physical address: 7845 Value: 0
Virtual address: 53159 Physical address: 935 Value: -23
Virtual address: 48725 Physical address: 17237 Value: 0
Virtual address: 2819 Physical address: 8451 Value: -64
Virtual address: 7316 Physical address: 10132 Value: 0
Virtual address: 45229 Physical address: 17581 Value: 0
Virtual address: 26859 Physical address: 17899 Value: 58
Virtual address: 28281 Physical address: 18041 Value: 0
Virtual address: 36161 Physical address: 2625 Value: 0
Virtual address: 1745 Physical address: 18385 Value: 0
Virtual address: 4553 Physical address: 14793 Value: 0
Virtual address: 55448 Physical address: 18584 Value: 0
Virtual address: 13617 Physical address: 18737 Value: 0
Virtual address: 64146 Physical address: 19090 Value: 62
Virtual address: 14228 Physical address: 19348 Value: 0
Virtual address: 36093 Physical address: 19709 Value: 0
Virtual address: 52193 Physical address: 8417 Value: 0
Virtual address: 15926 Physical address: 19766 Value: 15
Virtual address: 21548 Physical address: 11308 Value: 0
Virtual address: 19902 Physical address: 20158 Value: 19
Virtual address: 3676 Physical address: 15452 Value: 0
Virtual address: 32021 Physical address: 15637 Value: 0
Virtual address: 36948 Physical address: 20308 Value: 0
Virtual address: 43797 Physical address: 20501 Value: 0
Virtual address: 63781 Physical address: 20773 Value: 0
Virtual address: 47225 Physical address: 21113 Value: 0
Virtual address: 25355 Physical address: 21259 Value: -62
Virtual address: 44047 Physical address: 21519 Value: 3
Virtual address: 44382 Physical address: 21854 Value: 43
Virtual address: 60739 Physical address: 1091 Value: 80
Virtual address: 37563 Physical address: 22203 Value: -82
Virtual address: 46092 Physical address: 22284 Value: 0
Virtual address: 32587 Physical address: 14155 Value: -46
Virtual address: 44140 Physical address: 21612 Value: 0
Virtual address: 32113 Physical address: 15729 Value: 0
Virtual address: 40500 Physical address: 4660 Value: 0
Virtual address: 27863 Physical address: 22743 Value: 53
Virtual address: 55294 Physical address: 23038 Value: 53
Virtual address: 36441 Physical address: 5465 Value: 0
Virtual address: 47838 Physical address: 9182 Value: 46
Virtual address: 33025 Physical address: 10497 Value: 0
Virtual address: 50463 Physical address: 1311 Value: 71
Virtual address: 728 Physical address: 23256 Value: 0
Virtual address: 30826 Physical address: 23402 Value: 30
Virtual address: 43328 Physical address: 2112 Value: 0
Virtual address: 39683 Physical address: 23555 Value: -64
Virtual address: 575 Physical address: 23103 Value: -113
Virtual address: 40722 Physical address: 23826 Value: 39
Virtual address: 55190 Physical address: 22934 Value: 53
Virtual address: 37196 Physical address: 12364 Value: 0
Virtual address: 16567 Physical address: 24247 Value: 45
Virtual address: 36893 Physical address: 20253 Value: 0
Virtual address: 61892 Physical address: 24516 Value: 0
Virtual address: 36786 Physical address: 24754 Value: 35
Virtual address: 6018 Physical address: 24962 Value: 5
Virtual address: 55927 Physical address: 25207 Value: -99
Virtual address: 10444 Physical address: 9420 Value: 0
Virtual address: 42041 Physical address: 25401 Value: 0
Virtual address: 42158 Physical address: 25518 Value: 41
Virtual address: 12169 Physical address: 25737 Value: 0
Virtual address: 36641 Physical address: 24609 Value: 0
Virtual address: 15207 Physical address: 25959 Value: -39
Virtual address: 17683 Physical address: 26131 Value: 68
Virtual address: 44230 Physical address: 21702 Value: 43
Virtual address: 47933 Physical address: 26429 Value: 0
Virtual address: 42312 Physical address: 26696 Value: 0
Virtual address: 43014 Physical address: 5126 Value: 42
Virtual address: 11727 Physical address: 27087 Value: 115
Virtual address: 17732 Physical address: 26180 Value: 0
Virtual address: 63322 Physical address: 11866 Value: 61
Virtual address: 23746 Physical address: 13506 Value: 23
Virtual address: 29233 Physical address: 27185 Value: 0
Virtual address: 24405 Physical address: 27477 Value: 0
Virtual address: 4316 Physical address: 27868 Value: 0
Virtual address: 58993 Physical address: 6001 Value: 0
Virtual address: 62532 Physical address: 27972 Value: 0
Virtual address: 29568 Physical address: 16768 Value: 0
Virtual address: 40658 Physical address: 4818 Value: 39
Virtual address: 56189 Physical address: 28285 Value: 0
Virtual address: 22887 Physical address: 28519 Value: 89
Virtual address: 52321 Physical address: 28769 Value: 0
Virtual address: 64502 Physical address: 29174 Value: 62
Virtual address: 52614 Physical address: 7046 Value: 51
Virtual address: 50618 Physical address: 1466 Value: 49
Virtual address: 52889 Physical address: 29337 Value: 0
Virtual address: 2894 Physical address: 8526 Value: 2
Virtual address: 51689 Physical address: 29673 Value: 0
Virtual address: 52127 Physical address: 8351 Value: -25
Virtual address: 15957 Physical address: 19797 Value: 0
Virtual address: 22586 Physical address: 29754 Value: 22
Virtual address: 44725 Physical address: 30133 Value: 0
Virtual address: 11101 Physical address: 30301 Value: 0
Virtual address: 17499 Physical address: 30555 Value: 22
Virtual address: 3209 Physical address: 30857 Value: 0
Virtual address: 48405 Physical address: 30997 Value: 0
Virtual address: 30712 Physical address: 31480 Value: 0
Virtual address: 14340 Physical address: 31492 Value: 0
Virtual address: 18766 Physical address: 16974 Value: 18
Virtual address: 25687 Physical address: 31831 Value: 21
Virtual address: 55686 Physical address: 5766 Value: 54
Virtual address: 28535 Physical address: 32119 Value: -35
Virtual address: 38078 Physical address: 32446 Value: 37
Virtual address: 11410 Physical address: 32658 Value: 11
Virtual address: 26723 Physical address: 17763 Value: 24
Virtual address: 2712 Physical address: 32920 Value: 0
Virtual address: 54547 Physical address: 33043 Value: 68
Virtual address: 60563 Physical address: 33427 Value: 36
Virtual address: 52482 Physical address: 6914 Value: 51
Virtual address: 16837 Physical address: 33733 Value: 0
Virtual address: 3697 Physical address: 15473 Value: 0
Virtual address: 41621 Physical address: 13717 Value: 0
Virtual address: 4049 Physical address: 34001 Value: 0
Virtual address: 15362 Physical address: 34050 Value: 15
Virtual address: 32373 Physical address: 34421 Value: 0
Virtual address: 46155 Physical address: 22347 Value: 18
Virtual address: 42211 Physical address: 25571 Value: 56
Virtual address: 26697 Physical address: 17737 Value: 0
Virtual address: 38549 Physical address: 34709 Value: 0
Virtual address: 691 Physical address: 23219 Value: -84
Virtual address: 16794 Physical address: 33690 Value: 16
Virtual address: 14989 Physical address: 34957 Value: 0
Virtual address: 4782 Physical address: 35246 Value: 4
Virtual address: 20273 Physical address: 35377 Value: 0
Virtual address: 48410 Physical address: 31002 Value: 47
Virtual address: 27054 Physical address: 8110 Value: 26
Virtual address: 61800 Physical address: 24424 Value: 0
Virtual address: 64168 Physical address: 19112 Value: 0
Virtual address: 35268 Physical address: 11204 Value: 0
Virtual address: 7454 Physical address: 35614 Value: 7
Virtual address: 42098 Physical address: 25458 Value: 41
Virtual address: 51019 Physical address: 35915 Value: -46
Virtual address: 8237 Physical address: 36141 Value: 0
Virtual address: 53675 Physical address: 36523 Value: 106
Virtual address: 5385 Physical address: 36617 Value: 0
Virtual address: 5823 Physical address: 37055 Value: -81
Virtual address: 57084 Physical address: 16380 Value: 0
Virtual address: 64824 Physical address: 37176 Value: 0
Virtual address: 61300 Physical address: 3956 Value: 0
Virtual address: 34595 Physical address: 37411 Value: -56
Virtual address: 4193 Physical address: 27745 Value: 0
Virtual address: 11358 Physical address: 32606 Value: 11
Virtual address: 64895 Physical address: 37247 Value: 95
Virtual address: 44773 Physical address: 30181 Value: 0
Virtual address: 54698 Physical address: 33194 Value: 53
Virtual address: 17067 Physical address: 37803 Value: -86
Virtual address: 10202 Physical address: 38106 Value: 9
Virtual address: 40715 Physical address: 23819 Value: -62
Virtual address: 40169 Physical address: 38377 Value: 0
Virtual address: 18061 Physical address: 38541 Value: 0
Virtual address: 7599 Physical address: 35759 Value: 107
Virtual address: 682 Physical address: 23210 Value: 0
Virtual address: 40785 Physical address: 23889 Value: 0
Virtual address: 40259 Physical address: 38723 Value: 80
Virtual address: 32870 Physical address: 39014 Value: 32
Virtual address: 64533 Physical address: 39189 Value: 0
Virtual address: 3252 Physical address: 30900 Value: 0
Virtual address: 40644 Physical address: 4804 Value: 0
Virtual address: 37162 Physical address: 12330 Value: 36
Virtual address: 28839 Physical address: 39591 Value: 41
Virtual address: 10086 Physical address: 37990 Value: 9
Virtual address: 12643 Physical address: 39779 Value: 88
Virtual address: 4096 Physical address: 27648 Value: 0
Virtual address: 18128 Physical address: 38608 Value: 0
Virtual address: 5193 Physical address: 40009 Value: 0
Virtual address: 62532 Physical address: 27972 Value: 0
Virtual address: 27742 Physical address: 22622 Value: 27
Virtual address: 11454 Physical address: 32702 Value: 11
Virtual address: 29155 Physical address: 40419 Value: 120
Virtual address: 45679 Physical address: 40559 Value: -101
Virtual address: 38794 Physical address: 40842 Value: 37
Virtual address: 52685 Physical address: 7117 Value: 0
Virtual address: 56512 Physical address: 41152 Value: 0
Virtual address: 27729 Physical address: 22609 Value: 0
Virtual address: 34747 Physical address: 37563 Value: -18
Virtual address: 15739 Physical address: 41339 Value: 94
Virtual address: 8001 Physical address: 41537 Value: 0
Virtual address: 47345 Physical address: 21233 Value: 0
Virtual address: 18200 Physical address: 41752 Value: 0
Virtual address: 34690 Physical address: 37506 Value: 33
Virtual address: 63149 Physical address: 15277 Value: 0
Virtual address: 42808 Physical address: 42040 Value: 0
Virtual address: 42806 Physical address: 42038 Value: 41
Virtual address: 54076 Physical address: 42300 Value: 0
Virtual address: 45194 Physical address: 17546 Value: 44
Virtual address: 1675 Physical address: 18315 Value: -94
Virtual address: 28080 Physical address: 42672 Value: 0
Virtual address: 55524 Physical address: 18660 Value: 0
Virtual address: 1659 Physical address: 18299 Value: -98
Virtual address: 20916 Physical address: 42932 Value: 0
Virtual address: 27473 Physical address: 43089 Value: 0
Virtual address: 59387 Physical address: 43515 Value: -2
Virtual address: 29711 Physical address: 43535 Value: 3
Virtual address: 27518 Physical address: 43134 Value: 26
Virtual address: 41764 Physical address: 43812 Value: 0
Virtual address: 20273 Physical address: 35377 Value: 0
Virtual address: 11420 Physical address: 32668 Value: 0
Virtual address: 44192 Physical address: 21664 Value: 0
Virtual address: 36317 Physical address: 2781 Value: 0
Virtual address: 53909 Physical address: 3477 Value: 0
Virtual address: 44224 Physical address: 21696 Value: 0
Virtual address: 4685 Physical address: 35149 Value: 0
Virtual address: 62376 Physical address: 44200 Value: 0
Virtual address: 14381 Physical address: 31533 Value: 0
Virtual address: 4935 Physical address: 44359 Value: -47
Virtual address: 37949 Physical address: 32317 Value: 0
Virtual address: 29773 Physical address: 43597 Value: 0
Virtual address: 24450 Physical address: 27522 Value: 23
Virtual address: 21279 Physical address: 12575 Value: -57
Virtual address: 64825 Physical address: 37177 Value: 0
Virtual address: 11740 Physical address: 27100 Value: 0
Virtual address: 37540 Physical address: 22180 Value: 0
Virtual address: 53462 Physical address: 2006 Value: 52
Virtual address: 41725 Physical address: 13821 Value: 0
Virtual address: 47519 Physical address: 44703 Value: 103
Virtual address: 26269 Physical address: 44957 Value: 0
Virtual address: 18752 Physical address: 16960 Value: 0
Virtual address: 30904 Physical address: 23480 Value: 0
Virtual address: 58959 Physical address: 5967 Value: -109
Virtual address: 11618 Physical address: 26978 Value: 11
Virtual address: 45050 Physical address: 45306 Value: 43
Virtual address: 44750 Physical address: 30158 Value: 43
Virtual address: 56911 Physical address: 16207 Value: -109
Virtual address: 44821 Physical address: 45077 Value: 0
Virtual address: 41290 Physical address: 2378 Value: 40
Virtual address: 12439 Physical address: 45463 Value: 37
Virtual address: 9876 Physical address: 45716 Value: 0
Virtual address: 10311 Physical address: 9287 Value: 17
Virtual address: 34920 Physical address: 15976 Value: 0
Virtual address: 14798 Physical address: 46030 Value: 14
Virtual address: 28616 Physical address: 32200 Value: 0
Virtual address: 49427 Physical address: 46099 Value: 68
Virtual address: 42256 Physical address: 26640 Value: 0
Virtual address: 28246 Physical address: 18006 Value: 27
Virtual address: 48416 Physical address: 31008 Value: 0
Virtual address: 8427 Physical address: 36331 Value: 58
Virtual address: 24714 Physical address: 46474 Value: 24
Virtual address: 59850 Physical address: 46794 Value: 58
Virtual address: 25832 Physical address: 31976 Value: 0
Virtual address: 62515 Physical address: 27955 Value: 12
Virtual address: 35256 Physical address: 11192 Value: 0
Virtual address: 34845 Physical address: 15901 Value: 0
Virtual address: 2176 Physical address: 46976 Value: 0
Virtual address: 45167 Physical address: 17519 Value: 27
Virtual address: 32142 Physical address: 15758 Value: 31
Virtual address: 51243 Physical address: 13867 Value: 10
Virtual address: 33637 Physical address: 14437 Value: 0
Virtual address: 39044 Physical address: 47236 Value: 0
Virtual address: 62262 Physical address: 44086 Value: 60
Virtual address: 22874 Physical address: 28506 Value: 22
Virtual address: 64733 Physical address: 39389 Value: 0
Virtual address: 61018 Physical address: 47450 Value: 59
Virtual address: 24974 Physical address: 47758 Value: 24
Virtual address: 42469 Physical address: 26853 Value: 0
Virtual address: 8493 Physical address: 47917 Value: 0
Virtual address: 42422 Physical address: 26806 Value: 41
Virtual address: 16481 Physical address: 24161 Value: 0
Virtual address: 61213 Physical address: 3869 Value: 0
Virtual address: 16461 Physical address: 24141 Value: 0
Virtual address: 54862 Physical address: 8782 Value: 53
Virtual address: 39959 Physical address: 38167 Value: 5
Virtual address: 39867 Physical address: 23739 Value: -18
Virtual address: 26874 Physical address: 17914 Value: 26
Virtual address: 37458 Physical address: 22098 Value: 36
Virtual address: 62505 Physical address: 27945 Value: 0
Virtual address: 57022 Physical address: 16318 Value: 55
Virtual address: 32447 Physical address: 34495 Value: -81
Virtual address: 19549 Physical address: 48221 Value: 0
Virtual address: 40869 Physical address: 23973 Value: 0
Virtual address: 22668 Physical address: 29836 Value: 0
Virtual address: 18713 Physical address: 16921 Value: 0
Virtual address: 26956 Physical address: 8012 Value: 0
Virtual address: 44067 Physical address: 21539 Value: 8
Virtual address: 13173 Physical address: 48501 Value: 0
Virtual address: 36231 Physical address: 2695 Value: 97
Virtual address: 19124 Physical address: 48820 Value: 0
Virtual address: 16457 Physical address: 24137 Value: 0
Virtual address: 63330 Physical address: 11874 Value: 61
Virtual address: 11768 Physical address: 27128 Value: 0
Virtual address: 30657 Physical address: 31425 Value: 0
Virtual address: 63566 Physical address: 48974 Value: 62
Virtual address: 41946 Physical address: 43994 Value: 40
Virtual address: 46200 Physical address: 22392 Value: 0
Virtual address: 2144 Physical address: 46944 Value: 0
Virtual address: 51939 Physical address: 49379 Value: -72
Virtual address: 56364 Physical address: 41004 Value: 0
Virtual address: 37743 Physical address: 6255 Value: -37
Virtual address: 6516 Physical address: 49524 Value: 0
Virtual address: 47890 Physical address: 26386 Value: 46
Virtual address: 2766 Physical address: 32974 Value: 2
Virtual address: 59402 Physical address: 10 Value: 58
Virtual address: 33000 Physical address: 39144 Value: 0
Virtual address: 43884 Physical address: 20588 Value: 0
Virtual address: 28740 Physical address: 39492 Value: 0
Virtual address: 45466 Physical address: 49818 Value: 44
Virtual address: 5454 Physical address: 36686 Value: 5
Virtual address: 8578 Physical address: 48002 Value: 8
Virtual address: 12923 Physical address: 50043 Value: -98
Virtual address: 11315 Physical address: 32563 Value: 12
Virtual address: 24300 Physical address: 50412 Value: 0
Virtual address: 42552 Physical address: 50488 Value: 0
Virtual address: 63952 Physical address: 20944 Value: 0
Virtual address: 399 Physical address: 14991 Value: 99
Virtual address: 18628 Physical address: 50884 Value: 0
Virtual address: 6248 Physical address: 51048 Value: 0
Virtual address: 16595 Physical address: 24275 Value: 52
Virtual address: 17138 Physical address: 37874 Value: 16
Virtual address: 63984 Physical address: 20976 Value: 0
Virtual address: 36659 Physical address: 24627 Value: -52
Virtual address: 55676 Physical address: 5756 Value: 0
Virtual address: 36609 Physical address: 24577 Value: 0
Virtual address: 60131 Physical address: 51427 Value: -72
Virtual address: 10420 Physical address: 9396 Value: 0
Virtual address: 3905 Physical address: 33857 Value: 0
Virtual address: 54962 Physical address: 8882 Value: 53
Virtual address: 31577 Physical address: 51545 Value: 0
Virtual address: 19313 Physical address: 625 Value: 0
Virtual address: 26418 Physical address: 51762 Value: 25
Virtual address: 19738 Physical address: 19994 Value: 19
Virtual address: 58456 Physical address: 52056 Value: 0
Virtual address: 24602 Physical address: 46362 Value: 24
Virtual address: 7513 Physical address: 35673 Value: 0
Virtual address: 16415 Physical address: 24095 Value: 7
Virtual address: 34419 Physical address: 52339 Value: -100
Virtual address: 48562 Physical address: 31154 Value: 47
Virtual address: 2859 Physical address: 8491 Value: -54
Virtual address: 55524 Physical address: 18660 Value: 0
Virtual address: 45383 Physical address: 49735 Value: 81
Virtual address: 49594 Physical address: 46266 Value: 48
Virtual address: 13168 Physical address: 48496 Value: 0
Virtual address: 63156 Physical address: 15284 Value: 0
Virtual address: 10903 Physical address: 52631 Value: -91
Virtual address: 33382 Physical address: 10342 Value: 32
Virtual address: 26189 Physical address: 44877 Value: 0
Virtual address: 61733 Physical address: 24357 Value: 0
Virtual address: 52599 Physical address: 7031 Value: 93
Virtual address: 26462 Physical address: 51806 Value: 25
Virtual address: 16281 Physical address: 52889 Value: 0
Virtual address: 59800 Physical address: 46744 Value: 0
Virtual address: 23527 Physical address: 53223 Value: -7
Virtual address: 21981 Physical address: 6621 Value: 0
Virtual address: 2503 Physical address: 53447 Value: 113
Virtual address: 37184 Physical address: 12352 Value: 0
Virtual address: 30254 Physical address: 3118 Value: 29
Virtual address: 45343 Physical address: 49695 Value: 71
Virtual address: 28779 Physical address: 39531 Value: 26
Virtual address: 35911 Physical address: 19527 Value: 17
Virtual address: 8311 Physical address: 36215 Value: 29
Virtual address: 40707 Physical address: 23811 Value: -64
Virtual address: 18938 Physical address: 17146 Value: 18
Virtual address: 7301 Physical address: 10117 Value: 0
Virtual address: 30257 Physical address: 3121 Value: 0
Virtual address: 21294 Physical address: 12590 Value: 20
Virtual address: 13072 Physical address: 48400 Value: 0
Virtual address: 60140 Physical address: 51436 Value: 0
Virtual address: 49800 Physical address: 53640 Value: 0
Virtual address: 22196 Physical address: 53940 Value: 0
Virtual address: 3473 Physical address: 54161 Value: 0
Virtual address: 48797 Physical address: 17309 Value: 0
Virtual address: 8622 Physical address: 48046 Value: 8
Virtual address: 49240 Physical address: 54360 Value: 0
Virtual address: 39043 Physical address: 47235 Value: 32
Virtual address: 33281 Physical address: 10241 Value: 0
Virtual address: 38803 Physical address: 40851 Value: -28
Virtual address: 22216 Physical address: 53960 Value: 0
Virtual address: 48877 Physical address: 17389 Value: 0
Virtual address: 45229 Physical address: 17581 Value: 0
Virtual address: 27146 Physical address: 54538 Value: 26
Virtual address: 15137 Physical address: 25889 Value: 0
Virtual address: 44034 Physical address: 21506 Value: 43
Virtual address: 27153 Physical address: 54545 Value: 0
Virtual address: 64447 Physical address: 29119 Value: -17
Virtual address: 8195 Physical address: 36099 Value: 0
Virtual address: 62404 Physical address: 44228 Value: 0
Virtual address: 34976 Physical address: 16032 Value: 0
Virtual address: 44791 Physical address: 30199 Value: -67
Virtual address: 58632 Physical address: 54792 Value: 0
Virtual address: 14443 Physical address: 31595 Value: 26
Virtual address: 44925 Physical address: 45181 Value: 0
Virtual address: 17491 Physical address: 30547 Value: 20
Virtual address: 11581 Physical address: 26941 Value: 0
Virtual address: 23367 Physical address: 53063 Value: -47
Virtual address: 43871 Physical address: 20575 Value: -41
Virtual address: 49397 Physical address: 54517 Value: 0
Virtual address: 26417 Physical address: 51761 Value: 0
Virtual address: 9472 Physical address: 55040 Value: 0
Virtual address: 61130 Physical address: 47562 Value: 59
Virtual address: 40531 Physical address: 4691 Value: -108
Virtual address: 19622 Physical address: 48294 Value: 19
Virtual address: 51608 Physical address: 29592 Value: 0
Virtual address: 9648 Physical address: 55216 Value: 0
Virtual address: 60434 Physical address: 33298 Value: 59
Virtual address: 51951 Physical address: 49391 Value: -69
Virtual address: 14671 Physical address: 45903 Value: 83
Virtual address: 13219 Physical address: 48547 Value: -24
Virtual address: 8977 Physical address: 4113 Value: 0
Virtual address: 44144 Physical address: 21616 Value: 0
Virtual address: 27675 Physical address: 22555 Value: 6
Virtual address: 2478 Physical address: 53422 Value: 2
Virtual address: 63547 Physical address: 48955 Value: 14
Virtual address: 10818 Physical address: 52546 Value: 10
Virtual address: 59516 Physical address: 124 Value: 0
Virtual address: 31716 Physical address: 51684 Value: 0
Virtual address: 45256 Physical address: 17608 Value: 0
Virtual address: 53951 Physical address: 3519 Value: -81
Virtual address: 55672 Physical address: 5752 Value: 0
Virtual address: 41786 Physical address: 43834 Value: 40
Virtual address: 39608 Physical address: 55480 Value: 0
Virtual address: 41137 Physical address: 7601 Value: 0
Virtual address: 48001 Physical address: 26497 Value: 0
Virtual address: 19008 Physical address: 48704 Value: 0
Virtual address: 59793 Physical address: 46737 Value: 0
Virtual address: 6753 Physical address: 55649 Value: 0
Virtual address: 33712 Physical address: 14512 Value: 0
Virtual address: 15251 Physical address: 26003 Value: -28
Virtual address: 36994 Physical address: 20354 Value: 36
Virtual address: 63937 Physical address: 20929 Value: 0
Virtual address: 36786 Physical address: 24754 Value: 35
Virtual address: 30322 Physical address: 3186 Value: 29
Virtual address: 10729 Physical address: 56041 Value: 0
Virtual address: 16900 Physical address: 37636 Value: 0
Virtual address: 16484 Physical address: 24164 Value: 0
Virtual address: 44335 Physical address: 21807 Value: 75
Virtual address: 27190 Physical address: 54582 Value: 26
Virtual address: 52454 Physical address: 28902 Value: 51
Virtual address: 8250 Physical address: 36154 Value: 8
Number of Translated Addresses = 500
Page Faults = 219
Page Fault Rate = 0.438
TLB Hits = 38
TLB Hit Rate = 0.076
